src/components/molecules/StartTable/StartTable.scss

```diff
@@ -0,0 +1,45 @@
+@import "scss/constants.scss";
+
+$jazzbar-table-height: 189px;
+
+.StartTable {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  width: 100%;
+  background: $dark;
+  border: none;
+  border-radius: 22px;
+  color: $white;
+  margin: 10px 5px 5px;
```

Comment:
please, use descriptive variables in the file for spacing instead of just numbers.

Reply:
fixed in 849a156b9b81ba4db9c2e378d91ff47bc0551083